### ISSUE TITLE
Images that are not direct links do not load

### ISSUE THREAD

**Issue by fontune** [sansfont/Discord-Mirrorbot#2]
Images that would normally embed from a URL but are not direct links (for example tenor gifs) do not load.

**Comment by Emucr** [comment]
Are you still working on this? I think you need to support edited/deleted messages.

**Comment by fontune** [comment]
Agreed, to be honest this was more something I did for personal use but I’ll probably have a look into doing that when I get home from work. 

Was thinking about showing messages before and after edits for logging purposes and showing a 🗑 or ❌ emoji next to messages that are deleted.

**Comment by Emucr** [comment]
Are you planning on supporting reactions and pins also?

**Comment by fontune** [comment]
Again this is only something I did for personal use in my spare time, if I get round to it then yes.
